### DISCUSSION TITLE
doris 编译失败

### DISCUSSION THREAD

**Discussion by CrazyBeeline** [apache/doris#34574] [RFC]
os：openeuler22.03(sp3)
doris:2.0.9
编译步骤：
export DORIS_BUILD_PYTHON_VERSION="python3"
export DORIS_TOOLCHAIN="clang"
export REPOSITORY_URL="https://doris-thirdparty-repo.bj.bcebos.com/thirdparty"
export CUSTOM_NPM_REGISTRY="https://registry.npmmirror.com"

sh build.sh --fe --be --clean

错误信息：
![image](https://github.com/apache/doris/assets/25030234/da939f88-3e4b-49e7-be10-91215c79360f)

**Comment by yagagagaga** [comment]
Please use Docker for compilation.

```bash
docker pull apache/doris:build-env-for-2.0
```

**Comment by CrazyBeeline** [comment]
@yagagagaga  谢谢我试试
